ros/processor/insights_engine_result_consumer.py

```diff
@@ -130,7 +130,7 @@ def process_report(self, host, reports, utilization_info, performance_record):
                     number_of_recommendations=rec_count,
                     state=SYSTEM_STATES[state_key],
                     instance_type=performance_record.get('instance_type'),
-                    region=reports[0].get('details').get('region')
+                    region=performance_record.get('region')
                 )
```

Comment:
The idea of adding tests generally is to ensure that your changes are actually working.
Could you assert the `region` data in the system state tests i.e. along with the `instance_type`?

Reply:
`instance_type` assertion is already there. I assert the region data 👍🏻